region/lowernorfair/east/The Worst Room In The Game.json

```diff
@@ -207,8 +204,32 @@
           ]},
           {"heatFrames": 360}
         ]}
+      ]
+    },
+    {
+      "link": [1, 5],
+      "name": "Quick Pirate Avoid",
+      "requires": [
+        {"or": [
+          {"and": [
+            "canInsaneJump",
+            {"heatFrames": 110},
+            {"or": [
+              "canDownBack",
+              "Morph",
+              {"heatFrames": 40}
+            ]}
+          ]},
+          {"and": [
+            "canTrickyJump",
+            {"heatFrames": 200}
+          ]}
```

Comment:
What Eddie was doing, which may be the same as this: Stand on the left side of the 2 tile block, shoot down to break the block below, but not the one left of it. fall and hold left against the shot block that wasn't broken.

Reply:
I was trying to make trickyjump the more intuitive fall to the right and jump over the pirate, but doing it fast is really unrealistic.  Insanejump is left side fall, and now tricky jump is land on the platform and left side fall.